server/compute/diffexp_tiledb.py

```diff
@@ -1,9 +1,8 @@
-import os
 import concurrent.futures
```

Comment:
to be consistent with our naming, shouldn't the file be called `diffexp_cxg.py`?  Or perhaps `cxg_diffexp.py`?

Reply:
Done